src/mainwindow.h

```diff
@@ -24,14 +25,26 @@ class MainWindow : public QMainWindow
 {
 	Q_OBJECT
 
+private:
+	struct StatusbarItems {
+		QLabel* stepsKey;
+		QLabel* stepsValue;
+
+		// add more labels: "BoardId:" "702563:378 // 3:0:2:3:5:2:7 " hint: redPlanetQuadrant, .., .., .., permutation, board rotation, target cell
```

Comment:
Can you add the requested additional QLabels here,
you may also add your color ball here.
I'd also be fine with just a Label with empty space and the correct color.
Feel free to find your favourite solution.

Reply:
@r0ost3r Can you add the labels as requested in these comments?